subsys/bluetooth/controller/hci_driver.c

```diff
@@ -29,6 +29,12 @@ static K_SEM_DEFINE(sem_recv, 0, 1);
 static struct k_thread recv_thread_data;
 static K_THREAD_STACK_DEFINE(recv_thread_stack, CONFIG_SDC_RX_STACK_SIZE);
 
+/* The priority of RX thread should be always lower than SDC_HOST_SIGNAL_PRIO.
+ * It fetchs the hci events and acl data when there is no pending signal in SDC.
+ */
+#define SDC_HOST_SIGNAL_PRIO 4
+BUILD_ASSERT(SDC_HOST_SIGNAL_PRIO < CONFIG_SDC_RX_PRIO);
```

Comment:
What?

SDC_HOST_SIGNAL_PRIO is not used for anything else.

Reply:
SDC_HOST_SIGNAL_PRIO should be replaced by CONFIG_MPSL_THREAD_COOP_PRIO, but it introduces another discussion whether MPSL_THREAD_COOP_PRIO can be changed to 4 or not. I put it back to draft until the decision has been made.

Reply:
Ok, that makes more sense, although calling MPSL "host" is very misleading.

In any case I think we will probably get rid of SDC RX thread, and only have MPSL.
See here: https://github.com/nrfconnect/sdk-nrf/pull/3244

Reply:
Does it mean that hci_driver is going to process HCI events and ac data in a callback registered to mpsl? In such case, the callback is executed in signal_thread context with MPSL_THREAD_COOP_PRIO priority If I understand correctly.

Reply:
Correct.